lib/auth/types/openid/routes.js

```diff
@@ -129,9 +131,15 @@ module.exports = async function (pluginRoot, server, kbnServer, APP_ROOT, API_RO
                 try {
                     // Bell gives us the access token to identify with here,
                     // but we want the id_token returned from the IDP
-                    let {user} = await request.auth.securitySessionStorage.authenticate({
-                        authHeaderValue: 'Bearer ' + request.auth.artifacts['id_token']
-                    });
+                    if (credentials) {
+                        let {user} = await request.auth.securitySessionStorage.authenticate({
+                            authHeaderValue: 'Bearer ' + request.auth.artifacts['id_token'],
+                            refresh_token: request.auth.credentials.refreshToken
```

Comment:
Will the refresh token always here even when login for the first time?

Reply:
Yes